canaries/example/MyAmplifyApp/MyAmplifyAppApp.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  MyAmplifyAppApp.swift
+//  MyAmplifyApp
+//
+//  Created by Tsui, Yeung Wah on 4/12/22.
+//
+
+import SwiftUI
+import Amplify
+
```

Comment:
Yes, I can also try to import AmplifyPlugins, but to add the plugin before calling configureAmplify might be out of scope for this project as it will require a real configuration file